src/mutations/createAccount.js

```diff
@@ -94,7 +94,7 @@ export default async function createAccount(context, input) {
     // find all invites for this email address, for all shops, and add to all groups
     const emailAddresses = emails.map((emailRecord) => emailRecord.address.toLowerCase());
     invites = await AccountInvites.find({ email: { $in: emailAddresses } }).toArray();
-    groups = invites.map((invite) => invite.groupId);
+    groups = invites.reduce((allGroupIds, invite) => [...allGroupIds, ...invite.groupIds], []);
```

Comment:
You will need to add all `groupIds` AND the `groupId` if it exists, to support invites that were created before this PR was merged (unless you plan to add a migration to this PR to convert `groupId` to `groupIds`).

Also, I'm not sure whether it's possible to have duplicates here, but I'd feel better if you check for duplicates.

Maybe you can also update the `addAccountToGroup` call below to use `updateGroupsForAccounts` and not need the loop?